dictionary.lisp

```diff
@@ -0,0 +1,120 @@
+#|
+ This file is a part of Qtools-UI
+ (c) 2019 Shirakumo http://tymoon.eu (shinmera@tymoon.eu)
+ Author: Michał "phoe" Herda <phoe@disroot.org>
+|#
+
+(in-package #:org.shirakumo.qtools.ui)
+(in-readtable :qtools)
+
+(define-widget dictionary (qwidget)
+  ((empty-browser-text :accessor empty-browser-text
+                       :initarg :empty-browser-text)
+   (not-found-text :accessor not-found-text
+                   :initarg :not-found-text))
+  (:default-initargs
+    :empty-browser-text
+    "<i><p align=center>Type your query below and hit Search.</p></i>"
```

Comment:
This is not valid HTML. `p` is a block tag, and `i` is an inline tag, so the nesting should be switched.

Reply:
Fixed.